bootiso

```diff
@@ -1385,6 +1385,7 @@ function devi_configureLabel() {
     _vendor=$(lsblk -ldno VENDOR "$targetDevice" 2>/dev/null)
     _vendor=${_vendor:-FLASH}
     targetPartitionLabel=${targetPartitionLabel:-"${_user^^}_${_vendor^^}"}
+    targetPartitionLabel=${targetPartitionLabel:0:11}
```

Comment:
This should already be done here: https://github.com/jsamr/bootiso/blob/c217185be734fd1edece990da44e528d1621d176/bootiso#L1357

Reply:
it isnt working for some reasons.

Reply:
In that case I suggest to cope with this issue separately :-) Would you mind filling a bug report to help me track and reproduce the issue? I suggest you remove this commit from the PR, and I'll be happy to look into the doas feature.

Reply:
done!